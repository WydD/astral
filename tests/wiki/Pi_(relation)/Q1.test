<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<pi attributes="id,temp">
<source attributes="id,T,PHYID,temp" id="R" sourceAttributes="id,T,PHYID,temp" type="Relation"/>
</pi>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/wiki/Pi_(relation)/R.R" id="R" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
