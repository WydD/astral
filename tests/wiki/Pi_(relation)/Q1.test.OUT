id,temp
t=1000
1,27
1,27
1,29
t=1005
1,33
t=1010
1,25
1,31
1,33
t=1015
1,31
t=1020
t=1025
1,33
1,29
